openfisca_us/variables/usda/snap.py

```diff
@@ -9,14 +9,18 @@ class snap_gross_income(Variable):
     value_type = float
     entity = SPMUnit
     definition_period = YEAR
-    label = "SPM unit's gross income for calculating SNAP eligibility"
+    documentation = "SPM unit's gross income for calculating SNAP eligibility"
```

Comment:
```suggestion
    documentation = "Gross income for calculating SNAP eligibility"
```
We can remove all the entities from documentation